docs/integrations/common-workflow-steps/build-cache/cache-push.md

```diff
@@ -0,0 +1,73 @@
+
+With cache, you can persist any resource that is ignored by Git. So you can transfer files and folders between build pipelines. Sometimes it may speed up your build, or it may help if you have reliability issues with the original download location for dependencies. But keep in mind that the cache is uploaded to or downloaded from a remote location. It may help you in some cases, but it's not a guaranteed way to speed up builds. You should try and see the actual results of your project.
+
+The cache is stored as a single archive file. Cache push and pull components work in coordination on the same cache file defined with a label. With custom labeling, you can have different chunks of caches, and you can share some caches between branches. Cache labeling helps you organize your caches.
+
+When you drag and drop the cache push component into your workflow, it comes with pre-defined values according to your project type. For example, for Android projects, it comes with pre-defined gradle cache paths, which should be useful for most Android apps.
+
+If you need more paths to cache or need to change paths according to your project, you can customize included and excluded paths as you wish. All path updates will be reflected in the archived cache file on your next build.
+
+Cache push uses a pattern in order to select files and folders. Although the pattern is not a regexp, it's closer to a shell glob. For example, `~/Library/Caches/CocoaPods` and `./Pods` will select the "CocoaPods" folder from home and the repository direction as a whole. Or for an Android project, you can cache the ".gradle" folder with `~/.gradle` include path and exclude all ".lock" files from there with `~/.gradle/**/*.lock` exclude path. Patterns that can be used in both included and excluded paths are explained in detail [here](https://github.com/appcircleio/appcircle-cache-push-component#included--excluded-paths).
+
+:::warning
+
+Keep in mind that included paths and the cache push step's workflow order are closely related to each other. For example, if you include a path from a repository and you place the cache push step before the git clone step, cache push won't find that path since it's not git cloned yet. Although that's not a fatal error for cache push, it will inform you about unreachable paths on build logs. You can review and resolve those kinds of issues from build logs.
+
+:::
+
+You cannot reach the cache archive file directly by yourself. But you can see cache file updates and track changes to cache at the end of the build pipeline from "Download Artifacts > ac_cache.zip." Also, build logs have some useful information about the cache mechanism and how included and excluded paths are processed. You can see the produced cache file size from the build logs. (The size of the cache file affects upload and download durations.)
```

Comment:
Please add this under the **# Output Variable**. I know there is no output variable but you can change like that:
```suggestion
The Cache Push step does not have any output variables, and you cannot directly access the cache archive file. However, you can monitor cache file updates and track changes to the cache at the end of the build pipeline from "Download Artifacts > ac_cache.zip". Additionally, the build logs contain useful information about the cache mechanism and how included and excluded paths are processed. You can find details about the size of the produced cache file in the build logs. (The size of the cache file affects upload and download durations.)
```

Reply:
And it must have a warning about: if they want to use Download Artifacts they must have Export Build Artifacts step after this.